README.md

```diff
@@ -1,2 +1,132 @@
 # beaker-template
-Let's try and construct a Beaker DSL extension library!
+
+Let's construct a Beaker DSL extension library!
+
+# Why Would We Do Such a Thing?
+
+There are two reasons to create a Beaker library:
+
+1. To pull functionality out of Beaker core to be maintained/improved separately (mostly QE tasks)
```

Comment:
The text in parens is there to give people an idea of who this use case applies to.

I'd rather include too much info than too little.  Is there some reason it should be cut?